gcc/testsuite/rust/compile/cfg5.rs

```diff
@@ -0,0 +1,11 @@
+// { dg-additional-options "-w -frust-cfg=A=B" }
```

Comment:
Rustc expects `B` to be between double quotes.

Reply:
should we error our in this case then?

Reply:
Maybe?

Reply:
I've raised an issue from your comment we probably should do the same.